test/tls-provider.c

```diff
@@ -45,20 +74,23 @@ typedef struct xorkey_st {
     unsigned char pubkey[XOR_KEY_SIZE];
     int hasprivkey;
     int haspubkey;
+    char *tls_name;
+    _Atomic int references;
```

Comment:
Change `_Atomic int` to `CRYPTO_REF_COUNT`

Reply:
so changed in https://github.com/openssl/openssl/pull/19312/commits/f716ba4cebb9c0586b746e99554b6f92aada9501